packages/mui-base/src/Select/Select.tsx

```diff
@@ -332,7 +332,108 @@ Select.propTypes /* remove-proptypes */ = {
    */
   slotProps: PropTypes.shape({
     listbox: PropTypes.oneOfType([PropTypes.func, PropTypes.object]),
-    popper: PropTypes.oneOfType([PropTypes.func, PropTypes.object]),
+    popper: PropTypes.oneOfType([
```

Comment:
```suggestion
    popper: PropTypes /* @typescript-to-proptypes-ignore */.oneOfType([
```
I believe we should move this declaration here.